src/main/java/io/reactivex/plugins/RxJavaPlugins.java

```diff
@@ -231,7 +231,7 @@ public static Scheduler onSingleScheduler(Scheduler defaultScheduler) {
     }
 
     /**
-     * Removes all handlers and resets the default behavior.
+     * Removes all handlers and resets to default behavior.
```

Comment:
Again, not an expert in English, but "the" could be used here if there
would be something concrete to refer to. Or I'm wrong?

On Sun, 7 Aug 2016, 16:10 Niklas Baudy, <notifications@github.com> wrote:

> In src/main/java/io/reactivex/plugins/RxJavaPlugins.java
> <https://github.com/ReactiveX/RxJava/pull/4298#discussion_r73801630>:
>
> > @@ -231,7 +231,7 @@ public static Scheduler onSingleScheduler(Scheduler defaultScheduler) {
> >      }
> >
> >      /**
> > -     * Removes all handlers and resets the default behavior.
> > +     * Removes all handlers and resets to default behavior.
>
> how about resets to the default behavior.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ReactiveX/RxJava/pull/4298/files/879374dd3dd46429cc3541b5506c0098bc95e66e#r73801630>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA7B3CIszeYF28c2z_u-HhIqzrxP2s8Dks5qddlUgaJpZM4Jefje>
> .
>